repos/system_upgrade/el7toel8/actors/verifydialogs/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.models import DialogModel
+from leapp.reporting import Report
+from leapp.libraries.actor.library import check_dialogs
+from leapp.tags import ReportPhaseTag, IPUWorkflowTag
+
+
+class VerifyDialogs(Actor):
+    """
+    Check all dialogs and notify that user needs to make some choices.
+
+    Report messages containing all dialogs with questions that need to be answered will be produced.
+    """
+
+    name = 'verify_check_results'
+    consumes = (DialogModel,)
+    produces = (Report,)
+    tags = (ReportPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        check_dialogs(inhibit_if_no_userchoice=not self.skip_dialogs)
```

Comment:
Currently, when trying this on the test VM and running `leapp preupgrade` it finishes successfully with 0 exit code. Later, when running `leapp upgrade`, it fails informing users they did not provide any answer.

This probably contrast with what we wanted `preupgrade` to be, to communicate if the command passes, upgrade itself should be ok. What do you think @pirat89 ?

Reply:
We can't have interactivity during preupgrade stage. I can easily add an inhibitor and this will result in "smth went wrong" every time a preupgrade is run with the dialogs (literally every run on openstack rhel 7.6-updated image with authselect actors enabled). 
I decided not to stress out the user and go without an inhibitor for now. But it can be changed ofc.

Reply:
This is not much about interactivity I think. We were experiencing the same issue with one of our SSH actors where preupgrade always failed without explicit "PermitRootLogin yes".

I will have to take a look at the dialog again but it should not be difficult to add some workaround in our setup scripts (e.g vagrant, OpenStack), right?

Reply:
@fernflower As you can see, you already have there the ssh probably on major number of systems. What @Rezney is asking, it is not interactivity. The inhibitor should be raised definitely as in case that preupgrade ends with "green", you expect you can do an upgrade. but based on missing answers, it's not true. As well, when you answer everything, it can happen that the process will be inhibited based on anwer. We can prepare for people "elementary" answerfile (iternally only) and they could use it to skip this. Or we could put there default answers if this is really big pain. But the `preupgrade` must tell you already this information. There is not green with toxic waste hidden behind fence.

Reply:
@fernflower  But looking into the code, it looks that you wrote it well. It's exactly what I would expect, so I am now confused about the discussion at all (O_o). Did I miss something?

Reply:
I get your point, guys, and having slept on this I agree that you are right about the preupgrade idea of showing green report only if you are good to go with the upgrade.
I'll fix that in next commit.

Reply:
@fernflower thanks
@pirat89 Good idea with pre-crafted internal answerfile

Reply:
Done

Reply:
Review in progess.